.travis/config.sh

```diff
@@ -25,6 +25,12 @@
 #       - RUN_PYLINT_EXCLUDE
 #       - RUN_PYLINT_DISABLED
 #
+#   * .travis/runblack.sh:
+#
+#       - RUN_BLACK_INCLUDE
+#       - RUN_BLACK_EXCLUDE
+#       - RUN_BLACK_DISABLED
+#
```

Comment:
good! Can you please test this on the storage role? IIRC black made a huge list of changes for the storage role, so it would be good to disable it, at least in the beginning, so that the CI passes when those changes are integrated. And I believe @t-woerner will find it useful as well.